lib/httpClient.ts

```diff
@@ -43,4 +55,53 @@ export default abstract class HTTPClient {
             },
         );
     }
+
+    // https://github.com/axios/axios/issues/1666
```

Comment:
Could you add a comment explaining the problem/solution on why this issue is linked.